device/common/CMakeLists.txt

```diff
@@ -64,14 +64,12 @@ traccc_add_library( traccc_device_common device_common TYPE INTERFACE
    "include/traccc/finding/device/fill_sort_keys.hpp"
    "include/traccc/finding/device/make_barcode_sequence.hpp"
    "include/traccc/finding/device/propagate_to_next_surface.hpp"
-   "include/traccc/finding/device/prune_tracks.hpp"
```

Comment:
Please list the new `count_tracks` files instead of the removed `prune_tracks` ones.

Reply:
This no longer exists in the new version.